openapi/src/parts/tps.yaml

```diff
@@ -133,6 +133,24 @@ components:
           description: The policy object attempting to combine the conflicting policy
           $ref: '#/components/schemas/WsmPolicyDescription'
 
+    WsmPolicyMergeCheckResult:
+      type: object
+      required: [ conflicts, resourcesWithConflict ]
+      properties:
+        conflicts:
+          description: |
+            List of workspace conflicts.
+          type: array
+          items:
+            $ref: '#/components/schemas/WsmPolicyConflict'
+        resourcesWithConflict:
+          description: |
+            List of resources (by ID) in the target workspace with conflicting policy.
```

Comment:
curious about the schema here.

would it be helpful when the resourcesWithConflict contains an array of  resource uuid + the conflicts so we know exactly why resource A is a conflicted resources?

Reply:
I note that the caller has to test for two empty lists to decide if the check succeeded. Seems OK to me if it is OK with the UI folks.